src/PartitionedArraysExtensions.jl

```diff
@@ -0,0 +1,31 @@
+
+function generate_level_parts(parts,num_procs_x_level)
```

Comment:
Is the name of the file appropriate? It only contains the `generate_level_parts` function, which provides a very particular functionality required by multigrid methods. Not sure if this is something you would put in PartitionedArrays. Do you agree? If yes, where can we put it? BTW, is this function used by third party packages, or only within this package?